.github/workflows/build.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-18.04', 'ubuntu-20.04']
-        go: ['1.13', '1.14', '1.15', '1.16', '1.17']
+        go: ['1.15', '1.16', '1.17']
```

Comment:
Because of Firecracker Go SDK?

Reply:
* [go-openapi/runtime@0.23.0](https://github.com/firecracker-microvm/firecracker-go-sdk/commit/d6a80900f9228dc7a5d12f2cb4f6816d4347e369) does not work with go:1.13
* [golang.org/x/net](https://github.com/firecracker-microvm/firecracker-go-sdk/commit/bfa9fa8caac72864e08618ad2e5a7761826a5550) does not work with go:1.13 & 1.14